### DISCUSSION TITLE
.github not allowed as preset

### DISCUSSION THREAD

**Discussion by maxandersen** [renovatebot/renovate#27155] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

None

### Describe the problem

to avoid having unnnecessary extre repositories I put default.json in my .github repo.

But when I do:

```
{
  "extends": ["github>jbanghub/.github"]
}   
```

I get this error:

```
There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.

Error type: Cannot find preset's package (github>jbanghub/.github)

```

if i change it to:

```
{
  "extends": ["github>jbanghub/someothername"]
}   
```

it works.

how can I get renovate to use .github/default.json?

### Relevant debug logs

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by viceice** [comment, marked as answer]
are both repos public? it works well on renovate bot organization.

you should check the debug logs, especially the http stats

**Reply by maxandersen** [reply]
my error - had not fully pushed :/ so it all works now.
